src/PromiseCore.php

```diff
@@ -69,9 +69,9 @@ class PromiseCore
     /**
      * Create shared core.
      *
-     * @param RequestInterface $request HTTP request.
-     * @param resource         $handle cURL handle.
-     * @param ResponseBuilder  $responseBuilder Response builder.
+     * @param RequestInterface    $request HTTP request.
+     * @param resource|CurlHandle $handle cURL handle.
```

Comment:
```suggestion
     * @param resource|\CurlHandle $handle cURL handle.
```

or add CurlHandle to the `use` classes